Livestream: implement Pinned Comments

infinite-persistence opened the pull request:
## Issue
Fixes [#6420 Pinned Updates in SuperChats ](https://github.com/lbryio/lbry-desktop/issues/6420)

## Behavior
The top (latest) pinned comment will be displayed at the top of the livestream chat.  Only 1 pinned message will be displayed (same as other platforms).  When a comment is unpinned, the next pinned comment (if exists) will replace it's spot.

## Approach
- Livestream comments are retrieved from 2 ways:
    - From initial entry, `comment.List` is used to retrieved the chat.  Due to the recent pagination changes, the pinned comments will always be placed at the "top" (or in the chat GUI, the "bottom").
    - Subsequent messages will come via websocket. 
- We handle both cases by storing the pinned comment IDs in the reducer stage.

## Test
kp.odysee.com

tzarebczan commented:
This is working great, nice job KP! @jessopb feel free to ship after code review. 

In the future, it may be nice to allow them to edit the pinned comment - then they don't have to retype/pin a new one.

infinite-persistence commented:
### 2da7815 to aaf37be
- Corrected "Pinned" being inside the author name highligted bubble
- Fixed padding so that the message, if not scrolling, looks centered.

tzarebczan commented:
Just saw the comment about latest pinned being displayed if something is unpinned...that may be a bit strange in some cases for creators. 

I think updating the pin would mostly solve it - as they'd keep a single pin. We can rebroadcast the message into chat when a pin update is done (UI side or needs commentron?). Will file these ideas.

infinite-persistence commented:
> _Just saw the comment about latest pinned being displayed if something is unpinned...that may be a bit strange in some cases for creators.
I think updating the pin would mostly solve it - as they'd keep a single pin. We can rebroadcast the message into chat when a pin update is done (UI side or needs commentron?). Will file these ideas._

- Just a note:  YouTube's help page tells the user to delete and re-create. But I agree it's not user-friendly.
- #6858 - Can just copy/paste for now instead of refactor if needed quickly.
- Will also have to add something in the GUI to enforce a single pinned message per livestream.  
    - Unlikely event:  if creator decides to pin a comment via dev-tools Network, I think we would still end up with multiple pins.

infinite-persistence commented:
### 67c8b83 to 4731786 
- Rebased to resolve conflict
